.jenkins/jenkins.sh

```diff
@@ -75,7 +76,9 @@ echo "PYTHON env ${python_env}"
 # If the backend is a GTC backend we fetch the caches
 if [[ $backend != *numpy* ]];then
     echo "Fetching for exisintg gt_caches"
-    . ${JENKINS_DIR}/actions/fetch_caches.sh $backend $experiment
+    cd ${PACE_DIR}/fv3gfs-physics
+    . ${JENKINS_DIR}/fetch_caches.sh $backend $experiment
```

Comment:
pre-existing nit, but is there a reason this is executed with a leading `.`? The impact of this is the script gets run as if it's in the local namespace, i.e. writing any variables defined in fetch_caches.sh to the local namespace. I've had issues with this before overwriting SCRIPT_DIR to be the value from the called file.

On main there are no such references and it looks like this can be deleted.
```suggestion
    ${JENKINS_DIR}/fetch_caches.sh $backend $experiment
```